templates/part.content.php

```diff
@@ -21,6 +21,10 @@
 ?>
 <div id="surveyResults">
 
+	<?php if(empty($_['statistics'])) { ?>
+		No statistics available, please try later
```

Comment:
> *try again?

This implies that you should try again immediately which will not change the result. If no one reported any data yet you should try later, probably at least 24 hours because we build the statistics only once a day. But I don't think we need to be that precise. I just want to avoid to show a lot of empty tables.